frontend/src/data/api-service.ts

```diff
@@ -139,16 +138,14 @@ export const createApiService = ({
 
     /**
      * @param resource - The endpoint of the resource
-     * @param slug - the optional additional endpoint, used for collections.
      * @param params - Url parameter object
      * @returns response  The API response object
      */
     query<T = unknown>(
       resource: string,
-      slug: string = "",
       params: Record<string, string> = {}
     ): Promise<AxiosResponse<T>> {
-      return client.get(`${getResourceSlug(resource)}${slug}`, { params })
+      return client.get(`${getResourceSlug(resource)}`, { params })
```

Comment:
```suggestion
      return client.get(getResourceSlug(resource), { params })
```

No interpolation needed anymore